x-pack/plugins/apm/server/lib/helpers/get_tpm_rate.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SetupTimeRange } from './setup_request';
+
+export function getTpmRate({ start, end }: SetupTimeRange, eventCount: number) {
```

Comment:
Wouldn't `getThroughput` be more consistent with what we call it elsewhere?
```suggestion
export function getThroughput({ start, end }: SetupTimeRange, eventCount: number) {
```